packages/create-remix/templates/.eslintrc.js

```diff
@@ -1,8 +1,7 @@
 module.exports = {
   rules: {
-    "prefer-const": "warn",
-
     // because in our templates we shouldn't care
```

Comment:
Why not?

Reply:
It's user-facing code, so I think we _should_ care tbh.